src/test/utils/mocks/mocksContainer/mockText.ts

```diff
@@ -197,6 +197,10 @@ export default class MockText implements MockGameObject {
     this.color = color;
   });
 
+  setInteractive(hitArea, hitAreaCallback, dropZone) {
+    return null;
+  }
```

Comment:
Please use 
```suggestion

  setInteractive = vi.fn();
```

instead. That way the method is already a mock/spy and can be mocked/spied (obviously)

Reply:
Fixed, including in the other places mocking this function.